tests/example_wfs.py

```diff
@@ -2,8 +2,7 @@
 
 BASIC_WORKFLOW = """
 class: GalaxyWorkflow
-doc: |
-  Simple workflow that no-op cats a file and then selects 10 random lines.
+label: Simple workflow that no-op cats a file and then selects 10 random lines.
```

Comment:
So ``doc`` in CWL should be Galaxy's ``annotation`` not name I suppose?

Reply:
If so, would it be better to add a name here and keep this doc as the doc?

Reply:
Done in a rebase, where I also added copying of `doc` when converting from Format2 to abstract CWL.